README.md

```diff
@@ -124,6 +126,28 @@ If you want to setup the toolchain manually, you can set `rust-toolchain` to `ma
     rust-toolchain: manual
 ```
 
+### Build for target other than default
+
+By default, the action uses the default target (based on the host platform). You can force using a different target (which will be installed if missing) using `rust-target`:
+```yaml
+- name: Check semver
+  uses: obi1kenobi/cargo-semver-checks-action@v2
+  with:
+    rust-target: aarch64-apple-darwin
+```
+If you want to setup the toolchain manually, you can set `rust-toolchain` to `manual`. In this case, the action assumes the specified target is already installed and uses it:
+```yaml
+- name: Setup Rust
+  uses: dtolnay/rust-toolchain@master
+  with:
+    toolchain: stable 2 months ago
+- name: Check semver
+  uses: obi1kenobi/cargo-semver-checks-action@v2
+  with:
+    rust-toolchain: manual
+    rust-target: aarch64-apple-darwin
```

Comment:
Will this work as-is on the default Linux GitHub Actions runner?

I just want to make sure this example is copy-pasteable in its entirety, or else we wouldn't be setting up users for success.

Reply:
Good catch. I'll update it and give a test run just to be sure.